.github/workflows/sync-docs-full.yml

```
@@ -0,0 +1,87 @@
+name: Full Docs Sync to Vector Store
+
+on:
+  push:
+
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)